lib/containers/common.pm

```diff
@@ -210,6 +211,20 @@ sub test_container_image {
     }
 }
 
+sub test_image_zyp_ref_and_verify {
+    my ($image, $runtime) = @_;
+    # zypper lr
+    assert_script_run("$runtime run --rm $image zypper lr -s", 120);
+    # zypper ref
+    assert_script_run("$runtime run --name refreshed $image sh -c 'zypper -v ref | grep \"All repositories have been refreshed\"'", 120);
+    # Commit the image
+    assert_script_run("$runtime commit refreshed refreshed-image", 120);
+    # Remove it
+    assert_script_run("$runtime rm refreshed", 120);
+    # Verify the image works
+    assert_script_run("$runtime run --rm refreshed-image sh -c 'zypper -v ref | grep \"All repositories have been refreshed\"'", 120);
```

Comment:
Is `refreshed-image` used afterwards or can it be deleted?

Reply:
All the images are removed when the test ends from `clean_container_host`.